dts/arm/silabs/efm32_pg_1b.dtsi

```diff
@@ -155,6 +155,14 @@
                         interrupts = <2 0>;
                         status = "disabled";
                 };
+
+		pinctrl: pin-controller {
+			/* Pin controller is a "virtual" device since SiLabs SoCs do pin
+			 * control in a distributed way (GPIO registers and PSEL
+			 * registers on each peripheral).
+			 */
+			compatible = "silabs,gecko-pinctrl";
+		};
```

Comment:
This node seems to be a duplicate of an earlier one and causes the following build error:
```
devicetree error: Label 'pinctrl' appears on /soc/gpio@4000a400/pin-controller and on /soc/pin-controller
```